quarkchain/cluster/jsonrpc.py

```diff
@@ -1308,13 +1305,24 @@ async def getTotalSupply(self):
         return quantity_encoder(total_supply) if total_supply else None
 
     @public_methods.add
-    async def getTotalBalance(self):
-        # TODO: this is a mock implementation. using fake arguments for now
-        result = await self.master.get_total_balance(Branch(0x1), bytes(32), 0, None)
-        if not result:
-            raise None
-        total_balance, next_starter = result
-        return {"totalBalance": total_balance, "next": next_starter.hex()}
+    async def getTotalBalance(
+        self, branch, block_hash, token_id, starter=bytes(20), limit=100
+    ):
```

Comment:
Current I/O:
```json
{
"jsonrpc": "2.0",
"method": "getTotalBalance",
"params":["0x3",
	"0xe430c13890293dcea1d72fad39678af41d7d33cf25216b2a918b8fa9759e29f9",
	35760,
	"0x3d5b0f70a941c9d6b184825267646727ec2a99f1",
	50
	],
"id":14
}

{
  "jsonrpc": "2.0",
  "result": {
    "totalBalance": 50000000000000000000000000,
    "next": "0x5b06116f879c709921bdac65caea095c587d7e6e"
  },
  "id": 14
}
```

Reply:
to be consistent, integer input/output should be hex strings

Reply:
consider using `quantity_decoder`, address_decoder` and `hash_decoder`